float64[6] tcp_pose
---
bool success